### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by aditya-dl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,36 +5,30 @@
...
 
 - [Getting Started](#cvu--says-hi) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1FvebFw40Bm0bUHWCgS0-iuYp8AKLIfSh?usp=sharing)
-- [What and why is CVU?](#why-cvu-)
-- [Object Detection (YOLOv5)](#object-detection-yolov5) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1FvebFw40Bm0bUHWCgS0-iuYp8AKLIfSh?usp=sharing)
-  - [TensorRT](#tensorrt)
-  - [Torch](#torch)
-  - [ONNX](#onnx)
-  - [TensorFlow](#tensorflow)
-  - [TFLite](#tflite)
+- [What and why is CVU?](https://github.com/BlueMirrors/cvu/wiki)
```
Comment (thegrogu) [inline README.md:21]:
```suggestion
- [What is and why CVU?](https://github.com/BlueMirrors/cvu/wiki)
```
Even my suggestion might be wrong but ```Why is CVU ? ``` does not look correct. right?

Comment (thegrogu) [inline README.md:21]: Same suggestion here.

File: README.md [hunk 2]
```diff
@@ -44,9 +38,8 @@ CVU <img src="https://raw.githubusercontent.com/BlueMirrors/cvu/master/static/lo
 
 [Index](#index-)
 
-Whether you are developing an optimized computer vision pipeline or just looking to use some quick computer vision in your project, CVU <img src="https://raw.githubusercontent.com/BlueMirrors/cvu/master/static/logo.png" width="12"> can help! Designed to be used by both the expert and the novice, CVU <img src="https://raw.githubusercontent.com/BlueMirrors/cvu/master/static/logo.png" width="12"> aims at making CV pipelines easier to build and consistent around platforms, devices and models.<br>
 
-For example, how much installation-steps and code will you need to run object detection on a video with a TensorRT backend? How complicated is it be to test that pipeline in Colab?<br><br>
+How much installation-steps and code will you need to run object detection on a video with a TensorRT backend? How complicated is it be to test that pipeline in Colab?<br><br>
```
Comment (thegrogu) [inline README.md:42]:
```suggestion
How many installation-steps and lines of code will you need to run object detection on a video with a TensorRT backend? How complicated is it be to test that pipeline in Colab?<br><br>
```

File: README.md (hunk 1 above)
Comment (aditya-dl) [reply README.md:21]: Now that you mention it, I think it looks weird.

Comment (aditya-dl) [reply README.md:21]: Let's say we will change.

Comment (shivamswarnkar) [reply README.md]: How about we simply change it to "Introducing CVU"? Or just keep either "What is CVU" or "Why CVU".

File: README.md [hunk 3]
```diff
@@ -5,36 +5,30 @@
 
 
 <br>
+Computer Vision deployment tools for dummies and experts.<br><br>
```
Comment (shivamswarnkar) [inline README.md:8]:
```suggestion

Computer Vision deployment tools for dummies and experts.<br><br>
```

Comment (shivamswarnkar) [reply README.md:8]: Image alignment seems to be off without this extra line after ```<br>``` (checked on chrome and safari)

File: README.md [hunk 4]
```diff
@@ -44,9 +38,8 @@ CVU <img src="https://raw.githubusercontent.com/BlueMirrors/cvu/master/static/lo
 
 [Index](#index-)
 
-Whether you are developing an optimized computer vision pipeline or just looking to use some quick computer vision in your project, CVU <img src="https://raw.githubusercontent.com/BlueMirrors/cvu/master/static/logo.png" width="12"> can help! Designed to be used by both the expert and the novice, CVU <img src="https://raw.githubusercontent.com/BlueMirrors/cvu/master/static/logo.png" width="12"> aims at making CV pipelines easier to build and consistent around platforms, devices and models.<br>
 
-For example, how much installation-steps and code will you need to run object detection on a video with a TensorRT backend? How complicated is it be to test that pipeline in Colab?<br><br>
+How many installation-steps and lines of code will you need to run object detection on a video with a TensorRT backend? How complicated is it be to test that pipeline in Colab?<br><br>
 
 With CVU <img src="https://raw.githubusercontent.com/BlueMirrors/cvu/master/static/logo.png" width="12">, you just need the following! No extra installation steps needed to run on Colab, just pip install our tool, and you're all set to go!<br>
```
Comment (shivamswarnkar) [inline README.md:45]: Should we mention that we're using ```vidsz``` (and possibly add a link) in the demo, and it's optional?

Comment (makaveli10) [reply README.md:45]: Yes, I think its a good idea.. to get developers to stop coding open-cv boiler plate stuff

**Review by alanturing-bluemirrors — APPROVED** [review]

**Pull request merged**